src/core/mac/mac.cpp

```diff
@@ -659,48 +655,6 @@ void Mac::StartOperation(Operation aOperation)
 #endif
     }
 
-    switch (aOperation)
```

Comment:
Adding a note to help review: This whole `switch` is replaced by the `SetPending(aOperation);` above in the `if` block.